src/internal_rules.jl

```diff
@@ -696,11 +697,11 @@ function EnzymeRules.augmented_primal(config, func::Const{typeof(cholesky)}, RT:
         nothing
     else
         if EnzymeRules.width(config) == 1
-            Cholesky(Matrix(fact), 'L', 0)
+            Enzyme.make_zero(fact)
```

Comment:
AFAICT this will also create a `Cholesky` object with the same `uplo` char as `fact`? One issue I had noticed (but unfortunately had no time to open an issue) is that on main always `L` is used.

Reply:
correct this is what it would do